src/actinia_parallel_plugin/model/batch.py

```diff
@@ -0,0 +1,267 @@
+            'type': 'integer',
+            'description': 'Overall number of jobs within batchjob'
+            }
+        }
+
+
+class BatchProcessChainModel(Schema):
+    """Definition of the actinia-gdi batch process chain that includes several
+       actinia process chains that can be run in parallel or sequentially
+    """
+    type = 'object'
+    properties = {
+        'processing_host': {
+            'type': 'string',
+            'description': 'The actinia-core IP or URL in case the platform '
+                           'is not OpenShift and no new VM should be created '
+                           'by actinia-gdi'
+        },
+        'processing_platform': {
+            'type': 'string',
+            'description': 'The actinia-core platform, either "openshift" or '
+                           '"vm". If platform is "vm" and no actinia_core_url '
+                           'is given, actinia-gdi will create a new VM.'
+        },
+        'processing_platform_name': {
+            'type': 'string',
+            'description': 'The actinia-core platform name. Only used to '
+                           'match a job to a VM if VM not started by '
+                           'actinia-gdi. Ideally it would contain the job '
+                           'type (actinia-core-pt or actinia-core-oc) and '
+                           'a unique ID.'
+        },
+        'jobs': {
+            'type': 'array',
+            'items': ProcessChainModel,
+            'description': "A list of process chains (jobs) that should "
+                           "be executed in parallel or sequentially "
+                           "in the order provided by the list."
+        }
+    }
+    required = ["jobs"]
+
+
+class BatchJobResponseModel(Schema):
+    """Response schema for creating and requesting the status of a Batchjob
+    """
+    type = 'object'
+    properties = {
+        'actinia_core_jobid': {
+            'type': 'array',
+            'description': ('The actinia-core resource IDs for all individual '
+                            'jobs'),
+            'items': {'type': 'string'}
+        },
+        'actinia_core_platform': {
+            'type': 'string',
+            'description': ('The actinia-core platform, either "openshift"'
+                            ' or "vm"')
+        },
+        'actinia_core_platform_name': {
+            'type': 'string',
+            'description': 'The actinia-core platform name'
+        },
+        'actinia_core_response': {
+            'type': 'array',
+            'description': 'The responses of actinia-core for individual jobs',
+            'items': {'type': 'object'}
+        },
+        'actinia_core_url': {
+            'type': 'string',
+            'description': ('The actinia-core IP or URL where actinia-core '
+                            'is processing the batchjob')
+        },
+        'actinia_gdi_batchid': {
+            'type': 'integer',
+            'description': 'The batch ID'
```

Comment:
same here - we don't really need all of them

Reply:
`actinia_gdi_batchid` --> `batchid`